spec.md

```diff
@@ -527,24 +541,25 @@ message VolumeCapability {
 // `required_bytes` and `limit_bytes` can be set to the same value. At
 // least one of the these fields MUST be specified.
 message CapacityRange {
-  // Volume must be at least this big.
+  // Volume must be at least this big. This field is OPTIONAL.
+  // A value of 0 is equal to an unspecified field value.
   uint64 required_bytes = 1;
 
-  // Volume must not be bigger than this.
+  // Volume must not be bigger than this. This field is OPTIONAL.
+  // A value of 0 is equal to an unspecified field value.
   uint64 limit_bytes = 2;
 }
 
 // The information about a provisioned volume.
 message VolumeInfo {
   // The capacity of the volume in bytes. This field is OPTIONAL. If not
-  // set, it indicates that the capacity of the volume is unknown (e.g.,
-  // NFS share). If set, it MUST be non-zero.
+  // set (value of 0), it indicates that the capacity of the volume is
+  // unknown (e.g., NFS share).
   uint64 capacity_bytes = 1;
 
   // Contains identity information for the created volume. This field is
   // REQUIRED. The identity information will be used by the CO in
-  // subsequent calls to refer to the provisioned volume. This field
-  // should not exceed 1MiB.
```

Comment:
Hi @jdef,

I just want to be clear that the ID string previously would be allowed a field size of 1MiB, but according to the spec today the volume ID is now limited to 128 bytes. Is that correct?